flink-runtime/src/main/java/org/apache/flink/runtime/net/NetUtils.java

```diff
@@ -276,6 +289,13 @@ private static InetAddress findAddressUsingStrategy(AddressDetectionState strate
 				InetAddress interfaceAddress = ee.nextElement();
 
 				switch (strategy) {
+					case LOCAL_HOST:
```

Comment:
Why do we execute the `LOCAL_HOST` strategy for every interface? I thought we use `InetAddress.getLocalHost` as the outbound interface.